.gitignore

```diff
@@ -0,0 +1,7 @@
+build/
+_build/
+builddir/
```

Comment:
You don't need this. :)

https://mesonbuild.com/FAQ.html#how-do-i-ignore-the-build-directory-in-my-vcs

Meson sets it up automatically for you.

Reply:
It's to be sure that everything will be OK